extension/src/experiments/model/tree.ts

```diff
@@ -335,9 +345,30 @@ export class ExperimentsTree
     return [...this.view.selection]
   }
 
-  private getTooltip(error: string | undefined) {
+  private getTooltipTable(experiment: Experiment, firstThreeColumns: string[]) {
+    const data = firstThreeColumns
+      .map(path => {
+        const { value, splitUpPath } = getDataFromColumnPath(experiment, path)
+        const [type] = splitUpPath
+        return value
+          ? `| ${path.slice(type.length + 1) || path} | ${value} |\n`
+          : ''
+      })
+      .join('')
+    return getMarkdownString(`|First Three Columns||\n|:--|--|\n${data}`)
```

Comment:
Do we need a title for the tooltip? It takes up a lot of space and I'm not sure it's really that useful.

Reply:
Agreed! Took off the title :)